src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -24,63 +26,76 @@ const InfoLink = styled(ExternalLink)`
+      <AutoColumn gap="8px">
         <RowBetween>
-          <RowFixed align="center">
-            <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
-              {isExactIn ? 'Minimum received' : 'Maximum sold'}
-            </TYPE.body>
-            <QuestionHelper text="Your transaction will revert if there is a large, unfavorable price movement before it is confirmed." />
-          </RowFixed>
-          <RowFixed>
+          <RowBetween mr="13px">
             <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
-              {isExactIn
-                ? `${slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(4)} ${trade.outputAmount.currency.symbol}` ??
-                  '-'
-                : `${slippageAdjustedAmounts[Field.INPUT]?.toSignificant(4)} ${trade.inputAmount.currency.symbol}` ??
-                  '-'}
+              Max slippage
             </TYPE.body>
-          </RowFixed>
+            <RowFixed>
+              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
+                {formattedSlippage}%
+              </TYPE.body>
+              <StyledMenuIcon onClick={toggleSettings} id="open-settings-from-swap-dialog-button"></StyledMenuIcon>
+            </RowFixed>
+          </RowBetween>
+          <RowBetween ml="13px">
+            <RowFixed>
+              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
+                {isExactIn ? 'Min received' : 'Max sold'}
+              </TYPE.body>
+              <QuestionHelper text="Your transaction will revert if there is a large, unfavorable price movement before it is confirmed." />
+            </RowFixed>
+            <RowFixed>
+              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500" color="white">
+                {isExactIn
+                  ? `${slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(4)}` ?? '-'
+                  : `${slippageAdjustedAmounts[Field.INPUT]?.toSignificant(4)}` ?? '-'}
+              </TYPE.body>
+              <CurrencyLogo currency={currency} size="14px" style={{ margin: '0 2px' }} />
+            </RowFixed>
+          </RowBetween>
         </RowBetween>
         <RowBetween>
-          <RowFixed>
+          <RowBetween mr="13px">
+            <RowFixed>
+              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
+                Price impact
+              </TYPE.body>
+              <QuestionHelper text="The difference between the market price and estimated price due to trade size." />
+            </RowFixed>
+            <FormattedPriceImpact priceImpact={priceImpactWithoutFee} />
+          </RowBetween>
+          <RowBetween ml="13px">
+            <RowFixed>
+              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500">
+                Fees
+              </TYPE.body>
+              <QuestionHelper text="A portion of each trade (between 0% - 0.30%) goes to liquidity providers as a protocol incentive." />
```

Comment:
technically, the swap fee could be more than .3%

Reply:
Also, I'm not sure we should call it a "protocol" incentive as maybe that causes some confusion with the "protocol fee"

Reply:
Should be generalized here 128ac2f5519cf0a97ca4f6617c79e3d7693eecd7